Tests/L0/Common-MSBuildHelpers/Format-MSBuildArguments.AddsConfigurationProperty.ps1

```diff
@@ -5,6 +5,9 @@ param()
 . $PSScriptRoot\..\..\lib\Initialize-Test.ps1
 Microsoft.PowerShell.Core\Import-Module $PSScriptRoot\..\..\..\Tasks\MSBuild\ps_modules\MSBuildHelpers
 
+Unregister-Mock Get-UserAgentString
```

Comment:
Unregister is not required. Can you remove this line? Same comment applies to multiple files.